src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -239,7 +239,6 @@ template ClientCore ( )
             settings.conn_notifier, settings.auto_connect);
```

Comment:
>This method is de-facto deprecated and does nothing.

Mini comment: this is wrong. The method is _actually_ deprecated.

Reply:
Thanks, updated.